### ISSUE TITLE
Question about detached state

### ISSUE THREAD

**Issue by megamorf** [aaronjanse/3mux#70]
**Steps:**
1. connect to remote system via SSH and start 3mux on the remote machine
2. start long running task in that 3mux session
3. disconnect SSH connection

**Expected result:**
* 3mux session persists after the SSH session is closed and the long running task finishes.

**Actual result:**
* 3mux session dies and the long running task is killed prematurely.

Is the actual result to be expected? I'm used to screen and tmux where a detached session continues to run.

**Comment by aaronjanse** [comment]
> Is the actual result to be expected?

Not at all. The entire purpose of sessions should be lasting across SSH sessions. I'll look into this

**Comment by aaronjanse** [comment]
Thank you for the bug report!
